### PR TITLE
Windows build

### LINKED ISSUE #1: Windows Build
Raising as it would be great to have a Windows build. Are there specific issues that need to be addressed to achieve this?

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixes #1 

Please **do not merge** even if checks are green.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by aaronang** [comment]
@SylvainCorlay I just want to take the time to thank you for all your efforts :heart:

**Comment by SylvainCorlay** [comment]
OK, I got it. Will push a fix patching openexr cmake.

**Comment by SylvainCorlay** [comment]
So for the record, the cmake of openexr is one of the worst I ever encountered.

There are so many issues that it is hard to keep track

- The `openexr` cmake refers to the ilmbase headers from the source tarball even when building against an installed ilmbase.
- openexr libraries link with both the static and the shared versions of ilmbase libraries on windows.
- The FindOpenLibm.cmake gives the wrong visibility to the imported target making it impossible to link with them.
- ilmbase and openexr use each other's cmake options everywhere in the build, making it very difficult to separate.

and many more.

I patch the cmake in a very hacky way to get everything to build on windows, but this is not a proper fix for this. It should be completely rewritten in a more proper way.

File: recipe/0001-Imported-libraries-visibility.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
++        add_library(IlmBase::${ILMBASE_LIB} STATIC IMPORTED GLOBAL)
+         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES IMPORTED_LOCATION_RELEASE ${OPENEXR_${ILMBASE_LIB}_LIBRARY})
+         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES IMPORTED_LOCATION_DEBUG ${OPENEXR_${ILMBASE_LIB}_DEBUG_LIBRARY})
+         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES MAP_IMPORTED_CONFIG_RELWITHDEBINFO RELEASE)
+@@ -177,7 +177,7 @@ foreach(ILMBASE_LIB
+     endif()
+ 
+     if (OPENEXR_${ILMBASE_LIB}_STATIC_LIBRARY)
+-        add_library(IlmBase::${ILMBASE_LIB}_static SHARED IMPORTED)
++        add_library(IlmBase::${ILMBASE_LIB}_static STATIC IMPORTED GLOBAL)
```
Comment (jakirkham) [inline recipe/0001-Imported-libraries-visibility.patch]: It's really strange that they were using `SHARED` here. 😕  Did you try sending this patch to them?

**Comment by jakirkham** [comment]
Thanks for working on this @SylvainCorlay. 😄 LGTM

It would be good to submit the patch upstream. Seems like they would accept it.

File: recipe/0001-Imported-libraries-visibility.patch (hunk 1 above)
Comment (SylvainCorlay) [reply recipe/0001-Imported-libraries-visibility.patch]: No, I don't think that anyone is realistically using this.

Comment (jakirkham) [reply recipe/0001-Imported-libraries-visibility.patch]: Must not be. In any event, would be good to draw their attention to it.

**Pull request merged**